src/tag/constants.js

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+/* eslint-disable import/prefer-default-export */
+import DEFAULT_THEME from '../../src/themes/light-theme';
+
+export const STYLE = {
+  blue: DEFAULT_THEME.colors.primary470,
+  orange: DEFAULT_THEME.colors.warning400,
+  green: DEFAULT_THEME.colors.positive400,
+  red: DEFAULT_THEME.colors.negative400,
```

Comment:
I think we should name the props similar to the theme color naming: `primary, warning, positive, negative`, so it makes sence in the case, for example, when the primary color is set to `green`.

Reply:
ok